google/cloud/compute/v1/BUILD.bazel

```diff
@@ -217,6 +217,55 @@ java_gapic_assembly_gradle_pkg(
     ],
 )
 
+##############################################################################
+# Go
+##############################################################################
+load(
+    "@com_google_googleapis_imports//:imports.bzl",
+    "go_gapic_assembly_pkg",
+    "go_gapic_library",
+    "go_proto_library",
+    "go_test",
+)
+
+go_proto_library(
+    name = "compute_go_proto",
+    importpath = "google.golang.org/genproto/googleapis/cloud/compute/v1",
+    protos = [":compute_proto"],
+    deps = [
+        "//google/api:annotations_go_proto",
+    ],
+)
+
+go_gapic_library(
+    name = "compute_go_gapic",
+    srcs = [":compute_proto_with_info"],
+    grpc_service_config = "compute_grpc_service_config.json",
+    importpath = "cloud.google.com/go/compute/apiv1;compute",
+    service_yaml = "compute_v1.yaml",
+    transport = ["rest"],
```

Comment:
Is it already supported by the rules? If yes, then great, I just missed the moment when it was added.

Reply:
Yep, it has been supported for some time, it is just omitted from build_gen.